src/Apps/Order2/Routes/Details/Components/Order2DetailsFulfillmentInfo.tsx

```diff
@@ -0,0 +1,99 @@
+import { Box, Spacer, Text } from "@artsy/palette"
+import type { Order2DetailsFulfillmentInfo_order$key } from "__generated__/Order2DetailsFulfillmentInfo_order.graphql"
+import type React from "react"
+import { useFragment } from "react-relay"
+import { graphql } from "relay-runtime"
+
+interface Props {
+  order: Order2DetailsFulfillmentInfo_order$key
+}
+
+const getShippingContent = (fulfillmentDetails): React.ReactNode => {
+  const {
+    name,
+    addressLine1,
+    addressLine2,
+    city,
+    region,
+    postalCode,
+    country,
+    phoneNumber,
+  } = fulfillmentDetails
+  return (
+    <>
+      {name && (
+        <Text variant="xs" color="mono100">
+          {name}
+        </Text>
+      )}
+      {addressLine1 && (
+        <Text variant="xs" color="mono100">
+          {addressLine1}
+        </Text>
+      )}
+      {addressLine2 && (
+        <Text variant="xs" color="mono100">
+          {addressLine2}
+        </Text>
+      )}
+      {(city || region || postalCode) && (
+        <Text variant="xs" color="mono100">
+          {[city, region, postalCode].join(", ")}
```

Comment:
Assuming any of this can be blank, we probably need to compact the array to avoid something like ", , "?